sunkit_image/tests/test_stara.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import pytest
+from astropy.coordinates import SkyCoord
+from astropy.table import QTable
+from astropy.time import Time
+from sunpy.map import GenericMap
+
+from sunkit_image.stara import get_regions, stara
+from sunkit_image.tests.helpers import figure_test
+
+
+@pytest.fixture()
+def mock_map():
```

Comment:
We lack one of these?
If so, this should be moved to the main conftest.py file.

Reply:
My aim was to create a map with mock HMI data, I think the ones present in the conftest was AIA, though I haven't thoroughly looked in conftest, but I get the point, will check if it already has one.

Reply:
If you need a hmi one, you can move it there and rename it